app/queues/certificate/order-completer-worker.server.ts

```diff
@@ -88,6 +89,15 @@ export const orderCompleterWorker = new Worker<CertificateJobData>(
     });
 
     logger.info(`Certificate ${certificateId} successfully issued and stored`);
+    // send success email notification
+    logger.debug(
+      `Sending success notification email for certificate with id=${certificateEntry.id}, username=${certificateEntry.username}, domain=${certificateEntry.domain}`
+    );
+    await addNotification({
+      emailAddress: certificateEntry.user.email,
+      subject: 'My.Custom.Domain certificate ready',
+      message: `${certificateEntry.username}, your certificate with domain: ${certificateEntry.domain} is ready. Log in to My.Custom Domain to view/manage it.`,
```

Comment:
```suggestion
      message: `${certificateEntry.username}, your certificate with domain: ${certificateEntry.domain} is ready. Log in to My.Custom.Domain to view/manage it.`,
```
To make it same as subject

Reply:
Thanks for catching this!